logstash/repo.sls

```diff
@@ -1,15 +1,22 @@
+{%- from 'logstash/map.jinja' import logstash with context %}
+
 {%- if grains['os_family'] == 'Debian' %}
-logstash-repo-key:
-  cmd.run:
-    - name: wget -O - http://packages.elasticsearch.org/GPG-KEY-elasticsearch | apt-key add -
-    - unless: apt-key list | grep 'Elasticsearch (Elasticsearch Signing Key)'
+beats_repo_https_apt_support:
+  pkg.installed:
+    - name: apt-transport-https
 
-logstash-repo:
+logstash_repo:
```

Comment:
You use dashes in other state IDs, why this change here? In fact the state ID of the one being used for RedHat is `logstash-repo`